aircan/dependencies/google_cloud/bigquery_handler.py

```diff
@@ -31,9 +31,23 @@ def bq_import_csv(table_id, gcs_path, table_schema):
     
 def bq_schema_from_table_schema(table_schema):
     mapping = {
-        'number': 'float',
-        'year': 'integer'
-        }
+        'string': 'string',
+        "number": 'numeric',
+        "integer": 'numeric',
```

Comment:
@sagargg is it `numeric` for integers in Bigquery?

Reply:
@anuveyatsu It will parse both integer and float values.
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#numeric_type